docs/installation/install_epinio.md

```diff
@@ -137,3 +145,39 @@ To help you, see the following HowTos for various well-known Kubernetes clusters
 - [Install on EKS](../howtos/install_epinio_on_eks.md) - Install Epinio on AWS EKS cluster
 
 > *NOTE*: The Public Cloud howto lists the three major Cloud providers but Epinio can run on any Kubernetes cluster.
+
+## Internal Epinio components
+### Kubed
+
+Kubed is installed as a subchart when `.Values.kubed.enabled` is `true` (default).
+If you already have kubed running, you can skip the installation by setting
+the helm value `.Values.kubed.enabled` to `false`.
+
+### S3 storage
+
+Epinio is using an S3 compatible storage to store the application source code.
+This chart will install [Minio](https://min.io/) when `.Values.minio.enabled` is
+`true` (default).
+
+In addition to Minio, Epinio offers [s3gw](https://s3gw.io/) as another internal S3 compatible storage. It is installed when `.Values.minio.enabled` is set to `false` and `.Values.s3gw.enabled` is set to `true`.
+
+Both choices for internal S3 compatible storage can be configured to use a user-defined storageClass. If no StorageClass is defined, the default storageClass is used. For Minio the custom storageClass is set via the value of `.Values.persistance.storageClass`. For s3gw the custom storageClass is set via the value of `.Values.s3gw.storageClass.name`.
+
+Any external S3 compatible solution can be used instead by setting `.Values.minio.enabled` value to `false` (`.Values.s3gw.enabled` is `false` by default) and using [the values under `s3`](https://github.com/epinio/helm-charts/blob/b389a4875af9f03b484a911c49a14f834ba04b64/chart/epinio/values.yaml#L44) to point to the desired S3 server.
+
+### Dex
+
+[Dex](https://dexidp.io) OpenID Connect Provider is installed as a subchart when `.Values.global.dex.enabled` is set to `true` (default).
+
+In case you don't need to use an identity provider you may set the value to `false` and use only local epinio users. Here you will find more info about [OIDC Authentication](../references/authentication_oidc.md).
```

Comment:
```suggestion
In case you don't need to use an identity provider you may set the value to `false` and use only local epinio users. [OIDC Authentication](../references/authentication_oidc.md) provides more information.
```